requirements.txt

```diff
@@ -5,3 +5,5 @@ GitPython == 3.0.3
 PyStemmer == 1.3.0
 pyLDAvis == 2.1.2
 OpenNMT-py == 1.0.0rc1
+scikit-learn == 0.21.3
+seriate == 1.1.1
```

Comment:
Are we are going to use seriate?

Reply:
I plan to in following visualization. It was a bit premature though.